docs/tutorials/configure-external-signer-tls.md

```diff
@@ -22,9 +22,9 @@ This tutorial configures TLS between Teku and Web3Signer, and use the [`keytool`
 - [Web3Signer signing key files](https://docs.web3signer.consensys.net/en/latest/HowTo/Use-Signing-Keys/) for validators on the testnet.
 - [Teku Installed](../get-started/install/install-binaries.md).
 - [Java `keytool`](https://docs.oracle.com/en/java/javase/12/tools/keytool.html).
-- A running execution client such as [Hyperledger Besu], or cloud-based service such as [Infura] synced to the Goerli testnet.
+- A running execution client such as [Hyperledger Besu], or cloud-based service synced to the Goerli testnet.
 
-This tutorial connects to an ETH2 testnet, and uses [Infura] to access the Goerli ETH1 testnet. You can sign up for a free [Infura] account, or you can start your own [ETH1 Goerli node].
+This tutorial connects to an Ethereum testnet for the consensus layer and accesses the Goerli testnet for the execution layer. You can set up your own [Goerli node] for the execution layer. 
```

Comment:
Potentially also worth substituting in `Holesky` rather than `Goerli` (besu also has a run-a-node-on-holesky-testnet)

This tutorial creates an Ethereum node on the Holesky testnet. You can use any execution layer client, below references a besu guide.

Reply:
Thanks @rolfyone . I updated the docs accordingly. Please take a look